service/sign/handlers_sign.go

```diff
@@ -254,10 +255,22 @@ func Serial(w http.ResponseWriter, r *http.Request) response.ErrorResponse {
 	return response.ErrorResponse{Success: true}
 }
 
+// CleanHeader removes single quotes and leading and trailing white spaces from the header
+func CleanHeader(header string) string {
+	header = strings.Replace(header, "'", "", -1)
+	header = strings.TrimSpace(header)
+	return header
+}
+
 func checkRemodelingRequest(serialReq *asserts.SerialRequest, modelAssert, serialAssert asserts.Assertion, apiKey string) response.ErrorResponse {
-	originalBrandID := serialReq.HeaderString("original-brand-id")
-	originalModel := serialReq.HeaderString("original-model")
-	originalSerial := serialReq.HeaderString("original-serial")
+	originalBrandID := CleanHeader(serialReq.HeaderString("original-brand-id"))
```

Comment:
CleanHeader isn't really needed for the original brandID and original model as they are coming cleanly from the assertions

Reply:
Make sense, I removed all the not necessary changes

Reply:
Looks good